examples/tutorial/tutorial_notebooks/introduction/exercise_2.ipynb

```diff
@@ -517,7 +450,7 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.6.8"
+   "version": "3.8.12"
```

Comment:
Why do we exactly require 3.8.12?

Reply:
Also, replace `import pandas as old_pd` to just `import pandas` please in this file. We decided to use just `pandas` instead of `old_pd` in this [thread](https://github.com/modin-project/modin/pull/3705#discussion_r763695856).

Reply:
These changes automatically happen based on the last Python version run. I don't think we should worry about this change.